crates/holochain/Cargo.toml

```diff
@@ -75,6 +75,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_bytes = "0.11.12"
 serde_json = { version = "1.0.51", features = ["preserve_order"] }
 serde_yaml = "0.9"
+serde_with = { version = "3.12.0", features = ["json"] }
```

Comment:
Added `serde_with` for a simple way to modify the default serde serialization of a field within a struct.

If others would prefer to avoid the additional dependency I can implement a custom serializer instead.